Flabbergasted-Flamingos/bangazon-api
# Created Pull request template

# Description
This pull request is adding a pull_request_template

## Number of Fixes
No fixes, but will need to be tested that it works

## Related Ticket(s)
No related tickets, just initial repo set up

## Problem to Solve
Will give us formatting for pull requests to make sure we cover all the details.

## Proposed Changes
This will be added to master and once added will automatically pull up when we have pull requests.

## Expected Behavior
Once added to master should automatically pull up when we have pull_request_template

## Steps to Test Solution

1.  git fetch --all
2. git check out rb-pull-request-template
3. Read template and make sure it looks good
4. Actually I'm not sure how to test it without just adding it to master, but I'm probably just being pedantic right now.
